### DISCUSSION TITLE
Support iOS 18 Dark & Tinted Icons #auth

### DISCUSSION THREAD

**Discussion by kirbw** [ente-io/ente#3459] [Enhancements]
I currently have Ente Auth pinned to my home screen, but I'm using the new iOS18 tinted mode and the Auth icon sticks out like a sore thumb. 

See Apple's developer notes here. https://developer.apple.com/design/human-interface-guidelines/app-icons#iOS-iPadOS

**Comment by 0x7a-zip** [comment]
pleeeeeeeeease

**Comment by nnoooobbyy** [comment]
Been hoping for this for a while 🙏
